bin/download-logs

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative '../lib/maze'
+require_relative '../lib/maze/browser_stack_utils'
+require_relative '../lib/maze/logger'
+require 'net/http'
+require 'json'
+require 'uri'
+require 'fileutils'
+
+class DownloadLogEntry
+  def start(args)
+    @args = args.dup
+    options = args
+    username = ENV['BROWSER_STACK_USERNAME']
+    access_key = ENV['BROWSER_STACK_ACCESS_KEY']
```

Comment:
I've had to do it this way as I couldn't get the `parser`/`processor` to pick them up, even though they are set in the environment